docs/docs/extend/plugins/test.md

```diff
@@ -0,0 +1,44 @@
+---
+Title: Unit Tests
+---
+
+## Unit Tests
+For complicated plugins it makes sense to add unit tests to your code. InvenTree
```

Comment:
Maybe add something like "to ensure that plugins work correctly and are compatible with future versions too. You can run these tests as part of your ci against the current stable and latest tag to get notified when something breaks before it gets released as part of stable"???